src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -157,12 +157,14 @@ public ILogger CreateLogger(string name)
 #pragma warning disable CS0618
         private static void UpdateFormatterOptions(ConsoleFormatter formatter, ConsoleLoggerOptions deprecatedFromOptions)
         {
+            bool disableColors = deprecatedFromOptions.DisableColors || System.Console.IsOutputRedirected;
```

Comment:
Would it make sense to also be consistent with what Console does with regards to env vars?
https://github.com/dotnet/runtime/blob/e4ac6fa2e4f3de36cd32ed8790dee36ff0a7feaa/src/libraries/System.Console/src/System/ConsolePal.Unix.cs#L842-L879

Reply:
Thanks seems like this logic which enables color in console even when the output is redirected was added recently to fix this issue https://github.com/dotnet/runtime/issues/33980.

I'll include this to logging console as well